db_utils.py

```diff
@@ -95,15 +103,24 @@ def get_one_record(self, sql):
 
     def get_records(self, query):
         cursor = self.get_cursor()
-        cursor.execute(query)
+        try:
+            cursor.execute(query)
+        except Exception as e:
+            print(f"Exception thrown while processing sql: {query}\n{e}\n")
```

Comment:
this will fail on the next step, exception doesn't buy you anything here - you shoud re-raise it.
e.g:
try: 
  dangerous
except excveption as e:
  print diagnostic
  raise e